makefiles/package/deb.mk

```diff
@@ -10,7 +10,11 @@ THEOS_PLATFORM_DEB_COMPRESSION_LEVEL ?= 0
 endif
 
 _THEOS_DEB_PACKAGE_CONTROL_PATH := $(or $(wildcard $(THEOS_PROJECT_DIR)/control),$(wildcard $(THEOS_LAYOUT_DIR)/DEBIAN/control))
-_THEOS_DEB_CAN_PACKAGE := $(if $(_THEOS_DEB_PACKAGE_CONTROL_PATH),$(_THEOS_TRUE),$(_THEOS_FALSE))
+ifneq ("$(wildcard $(_THEOS_DEB_PACKAGE_CONTROL_PATH))","")
```

Comment:
This can remain `$(if …)` because empty string is equivalent to false.

Reply:
Wow did I ever overcomplicate things. Sorry, am still kinda new to make syntax. Switched back to the og syntax.

Reply:
That’s fine! Make is an awkward language I definitely don’t expect everyone to know all the tricks to it, totally fine if you write out whatever makes sense to you and I’ll give cleanup hints.